### ISSUE TITLE
Incorrect role is defined for "More Action...(Shift+F9)"button as menuitem: A11y_Azure Machine Learning Extension for VS Code_Settings_AI4D

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-tools-for-ai#2377]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags
#A11yTCS; #A11ySev2; #WCAG4.1.2; #BM_AzureMachineLearningExtensionforVSCode_Win32_May2024; #DesktopApp; #Azure Machine Learning Extension for VS Code; #A11yMAS; #Name Role Value; #Win11; #AI4D; 

### Environment Details:
Application Name: Azure Machine Learning Extension for VS Code
Visual Studio Code: Version 1.89.1(User setup)

### Repro Steps:

1. Install VS Code
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
4. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
5. In the search box, type "Azure Machine Learning"
6. Tab until the list of settings is active
7. Use the up/down arrows to navigate between settings
8. Tab till "More Action...shift+F9" button and Verify whether the correct role is defined for "More Action...(Shift+F9)"button or not.

### Actual Result:
Incorrect role is defined for "More Action...(Shift+F9)"button as menuitem.

### Expected Result:
Correct role should be defined for "More Action...(Shift+F9)" as button instead of menuitem.

### User Impact:
Screen reader users will face difficulty in accessing the control if the correct control type is not defined for the controls.

### Attachment:
![A11y_Azure Machine Learning Extension for VS Code_Settings_More Action_AI4D](https://github.com/microsoft/vscode-tools-for-ai/assets/98530069/1bce2430-3821-4730-ae7a-18b6301f3c7d)

**Comment by Yash14j** [comment]
GitHub Tags:#Rev:yaja;

**Comment by tbombach** [comment]
Thank you so much for filing this @kupatkar99!

This UI is provided by VS Code. Extensions define settings for their product and VS Code renders them in the settings editor (shown in the screenshot you provided). The "More Actions" button is rendered by VS Code and the role is set by that product. The guidelines suggest that I should re-assign this item back to you so you can file the bug for that product (please let me know if there is a different process I should follow). The VS Code repository is located here: https://github.com/microsoft/vscode

I will be happy to follow up with VS Code after the issue is filed to figure out the right fix for this.

**Comment by kupatkar99** [comment]
#External; #External:VSCode; #FirstParty:Bug214222;

External tracking ID: https://github.com/microsoft/vscode/issues/214222
